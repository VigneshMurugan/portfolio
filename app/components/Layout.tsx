export interface LayoutProps {}

export const Layout: React.FC<LayoutProps> = (_props) => {
  const { children } = _props;

  return (
    <div className="section-full flex flex-col">
      <div className="flex flex-1 items-center justify-center">{children}</div>
    </div>
  );
};
