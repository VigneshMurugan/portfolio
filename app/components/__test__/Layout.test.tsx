import { expect, test } from "vitest";
import { render } from "@testing-library/react";

import { Layout } from "../Layout";

describe("Layout Component", () => {
  const component = render(<Layout />);

  test("should render our Layout", () => {
    expect(component.baseElement).toMatchSnapshot();
  });
});
