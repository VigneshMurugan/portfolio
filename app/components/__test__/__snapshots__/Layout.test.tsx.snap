// Vitest Snapshot v1

exports[`Footer Component > should render our footer 1`] = `
<body>
  <div>
    <footer
      class="text-center text-sm"
    >
      <p>
        Built with 
        <span
          class="text-color-primary"
        >
          ♥
        </span>
         in
         
        <a
          href="https://www.surfline.com/surf-report/carlsbad-state-beach-north/5d4dc10680c2430001b119fb"
          rel="noreferrer"
          target="_blank"
        >
          San Diego
        </a>
        , California. © 
        2022
      </p>
    </footer>
  </div>
</body>
`;

exports[`Layout Component > should render our Layout 1`] = `
<body>
  <div>
    <div
      class="section-full flex flex-col"
    >
      <div
        class="flex flex-1 items-center justify-center"
      />
    </div>
  </div>
</body>
`;
